isis/src/base/objs/PvlObject/PvlObject.cpp

```diff
@@ -159,6 +159,9 @@ namespace Isis {
   void PvlObject::addLogGroup(Isis::PvlGroup &group) {
     addGroup(group);
     Application::Log(group);
+    QString msg = "This function(PvlObject::addLogGroup) will be depreciated in ISIS3 v9.0 in "
+                  "favor of Application::appendLogGroup";
+    std::cout << msg << std::endl;
```

Comment:
I feel ISIS has a better process for warnings? At the very least this should go to `cerr` instead of `cout`

Reply:
it should be redirected to cerr instead of cout now; this was just what Adam and I had come up with preliminarily.